doc/api/tls.md

```diff
@@ -876,6 +876,30 @@ socket.on('end', () => {
 });
 ```
 
+Passing in `net.Socket` will start a new instance.
+To upgrade an existing instance of `net.Socket` to a 
+`tls.TLSSocket`, pass it to `tls.connect()` as 
+the socket option: 
+
+```js
+const { Socket } = require('net');
+const tls = require('tls');
+const sock = new Socket();
```

Comment:
Uhm something went wrong here... The original code (see first commit) looked like `tls.connect({ socket: sock })`. @VerteDinde You probably don't want to use `tls.connect({ port: 6697, host: 'irc.freenode.net' })` in both cases, right? This won't upgrade an existing socket, it establishes a new connection.